src/V3Randomize.cpp

```diff
@@ -1424,6 +1432,7 @@ class CaptureVisitor final : public VNVisitor {
     AstVar* getVar(AstVar* const varp) const {
         const auto it = m_varCloneMap.find(varp);
         if (it == m_varCloneMap.end()) return nullptr;
+        if (it->second->origName().rfind("__Varg", 0) == 0) return nullptr;
```

Comment:
Please instead make a new var attribute flag, add it to the dump() and dumpJson, set it where this name is generated, then test it here.